tests/codeceptjs/core_test.js

```diff
@@ -114,29 +115,29 @@ Scenario('should change the form if form_name_root option is set @core', async (
 
 Scenario('should validate against oneOf schemas and display single oneOf and editors error messages @core @oneof', async (I) => {
   I.amOnPage('oneof.html')
-  I.waitForText('Object is missing the required property \'p4\'', '.alert-danger')
-  I.waitForText('Value must validate against exactly one of the provided schemas. It currently validates against 0 of the schemas.', '.alert-danger')
-  I.waitForText('Object is missing the required property \'p1\'', '.alert-danger')
-  I.waitForText('Object is missing the required property \'p2\'', '.alert-danger')
-  I.waitForText('Property must be set.', '[data-schemapath="root.p4"] .invalid-feedback')
-  I.waitForText('Property must be set.', '[data-schemapath="root.p5.p1"] .invalid-feedback')
-  I.waitForText('Property must be set.', '[data-schemapath="root.p5.p2"] .invalid-feedback')
+  I.waitForText('Object is missing the required property \'p4\'', DEFAULT_WAIT_TIME, '.alert-danger')
+  I.waitForText('Value must validate against exactly one of the provided schemas. It currently validates against 0 of the schemas.', DEFAULT_WAIT_TIME, '.alert-danger')
+  I.waitForText('Object is missing the required property \'p1\'', DEFAULT_WAIT_TIME, '.alert-danger')
+  I.waitForText('Object is missing the required property \'p2\'', DEFAULT_WAIT_TIME, '.alert-danger')
+  I.waitForText('Property must be set.', DEFAULT_WAIT_TIME, '[data-schemapath="root.p4"] .invalid-feedback')
+  I.waitForText('Property must be set.', DEFAULT_WAIT_TIME, '[data-schemapath="root.p5.p1"] .invalid-feedback')
+  I.waitForText('Property must be set.', DEFAULT_WAIT_TIME, '[data-schemapath="root.p5.p2"] .invalid-feedback')
   I.fillField('root[p4]', 'to')
   I.fillField('root[p5][p1]', 'to')
   I.fillField('root[p5][p2]', 'to')
   I.click('Get Value')
-  I.wait(3)
+  I.wait(DEFAULT_WAIT_TIME)
   I.dontSee('Object is missing the required property \'p4\'', '.alert-danger')
   I.dontSee('Object is missing the required property \'p1\'', '.alert-danger')
   I.dontSee('Object is missing the required property \'p2\'', '.alert-danger')
-  I.waitForText('Value must be at least 4 characters long.', '[data-schemapath="root.p4"] .invalid-feedback')
-  I.waitForText('Value must be at least 4 characters long.', '[data-schemapath="root.p5.p1"] .invalid-feedback')
-  I.waitForText('Value must be at least 4 characters long.', '[data-schemapath="root.p5.p2"] .invalid-feedback')
+  I.waitForText('Value must be at least 4 characters long.', DEFAULT_WAIT_TIME, '[data-schemapath="root.p4"] .invalid-feedback')
+  I.waitForText('Value must be at least 4 characters long.', DEFAULT_WAIT_TIME, '[data-schemapath="root.p5.p1"] .invalid-feedback')
+  I.waitForText('Value must be at least 4 characters long.', DEFAULT_WAIT_TIME, '[data-schemapath="root.p5.p2"] .invalid-feedback')
   I.fillField('root[p4]', 'todo')
   I.fillField('root[p5][p1]', 'todo')
   I.fillField('root[p5][p2]', 'todo')
   I.click('Get Value')
-  I.wait(3)
+  I.wait(DEFAULT_WAIT_TIME)
```

Comment:
Can we remove this "hard" wait?
If not please keep `3` here instead of a 10 second wait all the time.

Please also check other `I.wait` calls